docs/streams/developer-guide/dsl-api.html

```diff
@@ -3679,58 +3670,6 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
 <span class="c1">// Write the stream to the output topic, using explicit key and value serdes,</span>
 <span class="c1">// (thus overriding the defaults in the config properties).</span>
 <span class="n">stream</span><span class="o">.</span><span class="na">to</span><span class="o">(</span><span class="s">&quot;my-stream-output-topic&quot;</span><span class="o">,</span> <span class="n">Produced</span><span class="o">.</span><span class="na">with</span><span class="o">(</span><span class="n">Serdes</span><span class="o">.</span><span class="na">String</span><span class="o">(),</span> <span class="n">Serdes</span><span class="o">.</span><span class="na">Long</span><span class="o">());</span>
-</pre></div>
-                        </div>
-                        <p><strong>Causes data re-partitioning if any of the following conditions is true:</strong></p>
-                        <ol class="last arabic simple">
-                            <li>If the output topic has a different number of partitions than the stream/table.</li>
-                            <li>If the <code class="docutils literal"><span class="pre">KStream</span></code> was marked for re-partitioning.</li>
-                            <li>If you provide a custom <code class="docutils literal"><span class="pre">StreamPartitioner</span></code> to explicitly control how to distribute the output records
-                                across the partitions of the output topic.</li>
-                            <li>If the key of an output record is <code class="docutils literal"><span class="pre">null</span></code>.</li>
-                        </ol>
-                    </td>
-                </tr>
-                <tr class="row-odd"><td><p class="first"><strong>Through</strong></p>
```

Comment:
The diff is weird because the part above repeats below. The actual deletes starts here.